build-system/global-configs/canary-config.json

```diff
@@ -5,6 +5,6 @@
   "amp-live-list": 1,
   "amp-ios-overflow-x": 1,
   "amp-experiment": 1,
-  "expAdsenseA4A": 0.05,
-  "expDoubleclickA4A": 0.05
+  "expAdsenseA4A": 1.0,
```

Comment:
If you don't have a strong preference, maybe you could go ahead and deploy a new release based on this flag change and I'll make the update and submit a separate PR to go in for tomorrow's full release?

(I would have already, but I hit some git snag, and I'd rather not delay longer, while I untangle it, if possible.)